src/main/java/com/esri/samples/scene/display_scene/DisplaySceneSample.java

```diff
@@ -67,6 +67,11 @@ public void start(Stage stage) {
       // add a camera and initial camera position
       Camera camera = new Camera(28.4, 83.9, 10010.0, 10.0, 80.0, 0.0);
       sceneView.setViewpointCamera(camera);
+
```

Comment:
This should be removed

Reply:
good catch: didn't spot I'd accidentally done that to another file :(